README.md

```diff
@@ -73,7 +73,7 @@ We would love to have you contribute. For contributing guidance please refer to
 
 ## Documentation
 
-Visit the [Charmil Documentation](https://aerogear.github.io/charmil/docs/)
+Visit the [Charmil Documentation](https://github.com/aerogear/charmil/blob/main/starter/CONTRIBUTING.md#documentation)
```

Comment:
@KanishkKhurana I think you made changes to the wrong file. This link works perfectly.

As mentioned in issue #229, there's a broken link in this file: https://github.com/aerogear/charmil/blob/main/docs/src/intro.md#documentation.

Please make the required changes and ping me for review :)

Reply:
Really sorry for this , i will rectify this asap.